_pydevd_bundle/pydevd_utils.py

```diff
@@ -120,6 +125,56 @@ def quote_smart(s, safe='/'):
         return quote(s, safe)
 
 
+def get_clsname_for_code(code, frame):
+    clsname = None
+    if GC_SUPPORTED:
+        ## use of gc.get_referrers() was suggested by Michael Hudson
+        # all functions which refer to this code object
+        funcs = [f for f in gc.get_referrers(code)
```

Comment:
Yes, I like the idea to remove GC_SUPPORTED part. I've tested it and I haven't found any cases when it works better than inspecting `code.co_varnames`.